core/cas-server-core-web/src/main/java/org/apereo/cas/config/CasCoreWebConfiguration.java

```diff
@@ -30,6 +33,24 @@
     @Autowired
     private CasConfigurationProperties casProperties;
 
+    /**
+     * Get messages from one or more property files containing values
+     * that should be exposed to Thyme templates but don't need i18n.
+     * This bean is Autowired into {@link messageSource()}.
+     * @return PropertiesFactoryBean containing all common (non-i18n) messages
+     */
+    @Bean
+    public PropertiesFactoryBean casCommonMessages() {
```

Comment:
I like `common-messages.properties `. Definitely sounds better. As for the factory bean and logging, we probably could do both with `Throwables.propagate(e)` and that should take care of both the uncheckedness! of it as well as logging. 

If you wish to continue using the `PropertiesFactoryBean` and the bean that goes along with that, that's perfectly fine. In that case, we need to find a way to remove the auto-wiring  bit from the resource bundle itself. That's the bit I don't like. (We have a bean that gets everything set on it directly, and then somehow deep within itself gets an autowired field)

If I remember correctly, you can pass the factory bean as a parameter to the  bean that creates `CasReloadableMessageBundle`. Spring will auto-proxy the object from the factory for you. Example:

```java
public HierarchicalMessageSource messageSource(@Qualifier("casCommonMessagesFactoryBeanId") final Properties props)) {
   // set the props directly
}
```

That probably is the better compromise. See `JpaEventsConfiguration` as an example. 

Thanks.